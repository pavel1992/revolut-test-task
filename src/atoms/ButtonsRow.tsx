import styled from 'styled-components';

import { TABLET_BREAKPOINT } from '../styleConstants/mediaConstants';

export const ButtonsRow = styled.div`
  display: flex;
  flex-direction: row-reverse;
  margin: 24px 0;

  @media(max-width: ${TABLET_BREAKPOINT}) {
    margin-bottom: 0;
  }
`;
