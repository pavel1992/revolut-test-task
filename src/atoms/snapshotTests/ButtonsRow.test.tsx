import 'jest-styled-components';
import * as React from 'react';
import renderer from 'react-test-renderer';

import { ButtonsRow } from '../ButtonsRow';

test('renders ButtonsRow correctly', () => {
  const render = renderer
    .create(<ButtonsRow>111</ButtonsRow>)
    .toJSON();
  expect(render).toMatchSnapshot();
});
