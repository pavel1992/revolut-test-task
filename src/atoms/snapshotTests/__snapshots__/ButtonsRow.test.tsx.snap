// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ButtonsRow correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row-reverse;
  -ms-flex-direction: row-reverse;
  flex-direction: row-reverse;
  margin: 24px 0;
}

@media (max-width:768px) {
  .c0 {
    margin-bottom: 0;
  }
}

<div
  className="c0"
>
  111
</div>
`;
